pkg/transport/listener.go

```diff
@@ -91,6 +91,9 @@ type TLSInfo struct {
 	// Logger logs TLS errors.
 	// If nil, all logs are discarded.
 	Logger *zap.Logger
+
+	// If EmptyCN is true, cert must not have non empty Common Name.
+	EmptyCN bool
 }
```

Comment:
I think godoc will complain again needs to start with EmptyCN.

Reply:
ah this problem again... sorry for that, will fix soon